posts/2023-06-29-OutreachyInternshipExperience/index.qmd

```diff
@@ -0,0 +1,140 @@
+I want to express my heartfelt appreciation to my mentor, Jen Wokaty, who has been
+an exceptional guide, always steering me in the right direction whenever I faced
+challenges. Their mentorship has taught me invaluable lessons in problem-solving,
+finding joy in my work, and so much more. I am immensely grateful for the knowledge
+and skills I have acquired under their guidance.
+
+In this blog post, I want to share my enriching experience and the remarkable
+work I have been doing as part of the <a href="https://github.com/Bioconductor/sweave2rmd">sweave2rmd</a>
+project at Bioconductor. The sweave2rmd project focuses on converting vignettes
+from Sweave to R Markdown format, which plays a crucial role in modernizing the
+documentation of Bioconductor packages. Traditionally, the documentation has
+been written using a combination of LaTeX and R code, resulting in PDF files.
+By transforming these files into R Markdown format and rendering them as HTML files,
+we make the documentation more accessible, shareable, and interactive.
+
+Throughout my internship, I have made significant contributions to the
+sweave2rmd project. Some of my accomplishments include:
+
+- Converting numerous vignettes from Sweave to R Markdown format
+- Collaborating with maintainers to address issues and improve the conversion process
+- Taking the lead in maintaining the project board and tracking progress
+- Exploring automation processes to streamline the conversion workflow
+- Enhancing the project's documentation for better usability and clarity
+
+These experiences have not only honed my technical skills but have also allowed
+me to gain a deeper understanding of package development and documentation
+creation within the vibrant Bioconductor community. I'm proud to have submitted
+a total of 12 pull requests (PRs), out of which 8 have been successfully merged
+into Bioconductor in just one month. I am thrilled to continue making a
+meaningful impact on the project.
+
+If you're interested in reading more about my experience as an Outreachy intern
+with Bioconductor and my journey with the sweave2rmd project, I invite you to
+visit my [blog](https://dev.to/sonali8434). I share insights, challenges, and
+lessons learned along the way.
+
+I would like to express my sincere gratitude to Bioconductor, my mentors, and
+the entire community for providing me with this invaluable opportunity.
+Together, we are shaping the future of open science and making a difference in
+the world of scientific documentation.
+
+### DaenaRys Experience Working on the Microbiome Project
+
+I am a final-year computer science student with a love for computers and technology. 
+My fascination with biotechnology led me to choose the  <a href="https://www.outreachy.org/outreachy-may-2023-internship-cohort/communities/bioconductor/#optimize-microbiome-data-science-framework">"Optimizing Microbiome Data Science Framework"</a>
+project under Bioconductor. 
+
```

Comment:
I think rather than having a separate section about the project, it's better to include the paragraph for 109-120 here then you can remove the separate section about the project. We want a brief explanation of the project because this will be a long blog post.